src/operator/src/error.rs

```diff
@@ -850,116 +865,86 @@ impl ErrorExt for Error {
             | Error::ViewInfoNotFound { .. }
             | Error::TableNotFound { .. } => StatusCode::TableNotFound,
-
             Error::FlowNotFound { .. } => StatusCode::FlowNotFound,
-
             Error::JoinTask { .. } => StatusCode::Internal,
-
             Error::BuildParquetRecordBatchStream { .. }
             | Error::BuildFileStream { .. }
             | Error::WriteStreamToFile { .. }
             | Error::ReadDfRecordBatch { .. }
             | Error::Unexpected { .. } => StatusCode::Unexpected,
-
             Error::Catalog { source, .. } => source.status_code(),
-
             Error::BuildCreateExprOnInsertion { source, .. }
             | Error::FindNewColumnsOnInsertion { source, .. } => source.status_code(),
-
             Error::ExecuteStatement { source, .. }
             | Error::ExtractTableNames { source, .. }
             | Error::PlanStatement { source, .. }
             | Error::ParseQuery { source, .. }
             | Error::ExecLogicalPlan { source, .. }
             | Error::DescribeStatement { source, .. } => source.status_code(),
-
             Error::AlterExprToRequest { source, .. } => source.status_code(),
-
             Error::External { source, .. } => source.status_code(),
             Error::DeserializePartition { source, .. }
             | Error::FindTablePartitionRule { source, .. }
             | Error::SplitInsert { source, .. }
             | Error::SplitDelete { source, .. }
             | Error::FindRegionLeader { source, .. } => source.status_code(),
-
             Error::UnrecognizedTableOption { .. } => StatusCode::InvalidArguments,
-
             Error::ReadObject { .. }
             | Error::ReadParquetMetadata { .. }
             | Error::ReadOrc { .. } => StatusCode::StorageUnavailable,
-
             Error::ListObjects { source, .. }
             | Error::ParseUrl { source, .. }
             | Error::BuildBackend { source, .. } => source.status_code(),
-
             Error::ExecuteDdl { source, .. } => source.status_code(),
             Error::InvalidCopyParameter { .. } | Error::InvalidCopyDatabasePath { .. } => {
                 StatusCode::InvalidArguments
             }
-
             Error::ColumnDefaultValue { source, .. } => source.status_code(),
-
             Error::EmptyDdlExpr { .. }
             | Error::InvalidPartitionRule { .. }
             | Error::ParseSqlValue { .. }
             | Error::InvalidTimestampRange { .. } => StatusCode::InvalidArguments,
-
             Error::CreateLogicalTables { .. } => StatusCode::Unexpected,
-
             Error::ExecuteAdminFunction { source, .. } => source.status_code(),
             Error::BuildRecordBatch { source, .. } => source.status_code(),
-
             Error::UpgradeCatalogManagerRef { .. } => StatusCode::Internal,
             Error::StatementTimeout { .. } => StatusCode::Cancelled,
-
             Error::ColumnOptions { source, .. } => source.status_code(),
+            Error::Arrow { .. } => StatusCode::InvalidArguments,
```

Comment:
How is an arrow error status invalid argument?

Reply:
fixed  in 160c86dd6